geopandas/explore.py

```diff
@@ -204,6 +204,13 @@ def my_colormap(value):  # scalar value defined in 'column'
     highlight_kwds : dict (default {})
         Style to be passed to folium highlight_function. Uses the same keywords
         as ``style_kwds``. When empty, defaults to ``{"fillOpacity": 0.75}``.
+    missing_kwds : dict (default {})
+        Additional style for missing values (NaN):
+
+        color : str
+            Color of NaN values. Defaults to ``None``, which uses Folium's default.
+        label : str (default "NaN")
+            Legend entry for missing values.
```

Comment:
```suggestion
        Uses the same keywords as ``style_kwds``. Additionally, allows specification
        of the legend label.

        label : str (default "NaN")
            Legend entry for missing values.
```

It is not just color but any supported style.

Reply:
Thank you for your feedback, but I think it does not support the same keywords as `style_kwds`. All uses of `missing_kwds` I could find are:

- [`nan_color = missing_kwds.pop("color", None)`](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L554)
- `categories.append(missing_kwds.pop("label", "NaN"))` in [L682](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L682) and [L721](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L721)
- [`    m, "", [missing_kwds.pop("label", "NaN")], [nan_color]`](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L757)

Could you have a look again?